package.json

```diff
@@ -3,11 +3,10 @@
   "name": "open-in-atom",
   "description": "Open the current file in the Atom editor",
   "publisher": "sourcegraph",
-  "icon": "data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNDgiIGhlaWdodD0iNDgiIHZpZXdCb3g9IjAgMCA0OCA0OCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHJlY3QgeD0iNy4yMDAyIiB5PSI0Ljc5OTk5IiB3aWR0aD0iMTQuNCIgaGVpZ2h0PSIzLjYiIHJ4PSIxLjgiIGZpbGw9IiNCMjAwRjgiLz4KPHJlY3QgeD0iNy4yMDAyIiB5PSIzOC40IiB3aWR0aD0iMTQuNCIgaGVpZ2h0PSIzLjYiIHJ4PSIxLjgiIGZpbGw9IiNCMjAwRjgiLz4KPHJlY3QgeD0iMjEuNjAwNiIgeT0iMjEuNiIgd2lkdGg9IjE0LjQiIGhlaWdodD0iMy42IiByeD0iMS44IiBmaWxsPSIjMDBCNEYyIi8+CjxyZWN0IHg9IjI0IiB5PSI0Ljc5OTk5IiB3aWR0aD0iOS42IiBoZWlnaHQ9IjMuNiIgcng9IjEuOCIgZmlsbD0iIzAwQjRGMiIvPgo8cmVjdCB4PSIxNC40MDA0IiB5PSIzMCIgd2lkdGg9IjE0LjQiIGhlaWdodD0iMy42IiByeD0iMS44IiBmaWxsPSIjMDBCNEYyIi8+CjxyZWN0IHg9IjMxLjIwMDIiIHk9IjMwIiB3aWR0aD0iOS42IiBoZWlnaHQ9IjMuNiIgcng9IjEuOCIgZmlsbD0iI0Y5NjIxNiIvPgo8cmVjdCB4PSIzOC40MDA0IiB5PSIyMS42IiB3aWR0aD0iNy4yIiBoZWlnaHQ9IjMuNiIgcng9IjEuOCIgZmlsbD0iI0Y5NjIxNiIvPgo8cmVjdCB4PSIxNC40MDA0IiB5PSIxMy4yIiB3aWR0aD0iMTQuNCIgaGVpZ2h0PSIzLjYiIHJ4PSIxLjgiIGZpbGw9IiMwMEI0RjIiLz4KPHJlY3QgeD0iMzEuMjAwMiIgeT0iMTMuMiIgd2lkdGg9IjcuMiIgaGVpZ2h0PSIzLjYiIHJ4PSIxLjgiIGZpbGw9IiNGOTYyMTYiLz4KPHJlY3QgeD0iMjQiIHk9IjM4LjQiIHdpZHRoPSI0LjgiIGhlaWdodD0iMy42IiByeD0iMS44IiBmaWxsPSIjRjk2MjE2Ii8+Cjwvc3ZnPgo=",
   "activationEvents": [
     "*"
   ],
-  "wip": false,
+  "wip": true,
```

Comment:
**question**: Why WIP? What's missing?

Reply:
Nothing I can think of, just didn't want to take it out of WIP before review